shame_command.py

```diff
@@ -0,0 +1,80 @@
+import configparser
+import logging
+import os
+import aiohttp
+import discord
+from discord.ext import commands
+
+logger = logging.getLogger(__name__)
+
+# Load the config file
+config_path = os.path.join(os.path.dirname(__file__), "settings.cfg")
+config = configparser.ConfigParser()
+config.read(config_path)
+
+TODOIST_API = "https://api.todoist.com/rest/v2/tasks"
+
+
+# Retrieve Todoist API Key for the email from the config file
+def get_todoist_token(email: str) -> str:
+    config.read(config_path)
+    if email in config["TODOIST_KEY_BY_EMAIL"]:
+        return config["TODOIST_KEY_BY_EMAIL"][email]
+    else:
+        return None
+
+
+# Get tasks from Todoist API with 'shame' label using aiohttp
+async def get_shame_tasks(todoist_token: str, session: aiohttp.ClientSession) -> list:
```

Comment:
I feel this is a good time to extract all the todoist logic to a separate method, sharing similar functions like this one?

also `list` is insufficient typing

Reply:
Changed list -> List[Task]

sounds good to refactor as separate change